src/pybind/mgr/dashboard/services/exception.py

```diff
@@ -51,6 +52,9 @@ def dashboard_exception_handler(handler, *args, **kwargs):
         cherrypy.response.headers['Content-Type'] = 'application/json'
         cherrypy.response.status = getattr(e, 'status', 400)
         return json.dumps(serialize_dashboard_exception(e)).encode('utf-8')
+    except Exception as exc:
```

Comment:
```suggestion
    except Exception as e:
```
As the previous handler also calls it `e`.

Reply:
@epuertat As Clean Code encourages meaningful names:
https://dzone.com/articles/naming-conventions-from-uncle-bobs-clean-code-phil

Then I'll rename the previous handler also.

Reply:
Up to you, but in some contexts (short scoped, well-known) one-letter variable name is ok (as in this case, or `for i in collection`, `for k,v in dict`...). In Python perhaps `ex` is more used than just `e` (in fact, pylint accepts `e` as an exception for a good variable name).